packages/url_launcher/url_launcher/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Updates minimum supported SDK version to Flutter 3.19/Dart 3.3.
```

Comment:
@mdebbar @harryterkelsen Do we need to add something about the recent a11y fixes?

Reply:
The a11y fixes are still in master, not in a stable release yet. Not sure what you had in mind here.